internal/controller/storageclaim_controller.go

```diff
@@ -523,68 +522,20 @@ func (r *StorageClaimReconciler) getCephRBDStorageClass(data map[string]string)
 	return storageClass
 }
 
-func (r *StorageClaimReconciler) getCephFSVolumeSnapshotClass(data map[string]string) *snapapi.VolumeSnapshotClass {
+func (r *StorageClaimReconciler) getCephDriverVolumeSnapshotClass(
+	labels map[string]string, annotations map[string]string) *snapapi.VolumeSnapshotClass {
 	volumesnapshotclass := &snapapi.VolumeSnapshotClass{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: r.storageClaim.Name,
+			Name:        r.storageClaim.Name,
+			Labels:      labels,
+			Annotations: annotations,
```

Comment:
getCephDriverVolumeSnapshotClass is only supported for cephfs not for RBD? the Driver name is hardcoded to cephfs driver.

Reply:
Yeah, I left out some of the code now it's back